src/box/iproto.cc

```diff
@@ -500,6 +500,19 @@ struct iproto_connection
 	 * connection.
 	 */
 	struct cmsg destroy_msg;
+	/**
+	 * Wait for shutdown readiness and close connection.
+	*/
+	struct ev_async after_shutdown;
+	/** Happened shutdown(socket, SHUT_RD) on input socket. */
+	bool socket_shutdown;
+	/**
+	* Connection support graceful shutdown. Is set only after start of
+	* shutdown process.
```

Comment:
The comment is misaligned. `*` should be right below first `*` of the first line.